File: problemas/1100/1132.php
<?php

$x = readline();
$y = readline();
$menor = min($x, $y);
$maior = max($x, $y);
$soma = 0;

for ($i = $menor; $i <= $maior; $i++) {
    if ($i % 13 != 0) {
        $soma += $i;
    }
}

echo $soma . "\n";

File: problemas/1100/1132.cs
using System;

namespace uri1131
{
    internal static class Program
    {
        private static void Main()
        {
            int x = int.Parse(Console.ReadLine());
            int y = int.Parse(Console.ReadLine());
            int menor = Math.Min(x, y);
            int maior = Math.Max(x, y);
            int soma = 0;

            for (int i = menor; i <= maior; i++)
            {
                if (i % 13 != 0)
                {
                    soma += i;
                }
            }

            Console.WriteLine(soma);
        }
    }
}